.env.example

```diff
@@ -0,0 +1,6 @@
+SERVER_API=https://peli.sketchni.uk/
+SERVER_KEY=
```

Comment:
What are SERVER_KEY and CLIENT_KEY?

Reply:
They could be better named. They're the application api key and client api keys generated in Pelican/Pterodactyl

Reply:
Got it - I forget that you have separate API tokens for both. Rename SERVER_ to APPLICATION_ and it should be ok?